sdk/tests/deqp/functional/gles3/es3fNegativeTextureApiTests.js

```diff
@@ -1674,21 +1674,21 @@ goog.scope(function() {
         function() {
             bufferedLogToConsole('gl.INVALID_ENUM is generated if target or pname is not one of the accepted defined values.');
             gl.texParameteri(0, gl.TEXTURE_MIN_FILTER, gl.LINEAR);
-            this.expectError(gl.INVALID_ENUM);
+            this.expectError([gl.INVALID_ENUM, gl.INVALID_OPERATION]);
```

Comment:
Rather than deviate from the behavior of the native tests, which only expect INVALID_ENUM here, could you please instead modify this test to create and bind a texture object?